client/src/components/SideTab/SideTab.css

```diff
@@ -0,0 +1,4 @@
+.isSelected {
```

Comment:
Try to use same convention for naming out the class names. https://en.bem.info/methodology/naming-convention/

Reply:
I agree with you. We haven't yet finalized/documented what conventions we should follow for custom CSS. Thanks for the reference, I will propose to have a project to have our rulebook for CSS conventions.